client/src/components/Overview.jsx

```diff
@@ -86,13 +120,16 @@ const OverviewContainer = connect(
   (state) => ({
     productInfo: state.productInfo,
     styles: state.styles,
-    selectedStyle: state.selectedStyle
+    selectedStyle: state.selectedStyle,
+    metaData: state.metaData
   }),
   // links the event handler to the store via dispatch
   (dispatch) => ({
+    // eslint-disable-next-line object-shorthand
```

Comment:
Really clean incoroporation and organization of all the different sections. the container attributes are super well organizes and the bottom and the layout makes your code really easy to read